apps/studio/src/server/modules/site/site.router.ts

```diff
@@ -51,7 +52,11 @@ export const siteRouter = router({
     .input(setNotificationSchema)
     .mutation(async ({ input, ctx }) => {
       const { siteId, notification } = input
-      await setSiteNotification(siteId, notification)
+      if (notification) {
```

Comment:
since you have standardized schema, you can probably use `notificationEnabled` boolean for this instead of a falsey check